src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -633,8 +645,10 @@ class ParseLiveQueryServer {
       };
       const trigger = getTrigger('@Connect', 'beforeConnect', Parse.applicationId);
       if (trigger) {
-        const auth = await this.getAuthForSessionToken(req.sessionToken);
-        req.user = auth.user;
+        const { auth } = await this.getAuthForSessionToken(req.sessionToken);
```

Comment:
Can we deduplicate here too? Seems like the other method that you created to deduplicate could serve this one as well, as it contains the exact same code?